spec/forms/teacher_interface/alternative_name_form_spec.rb

```diff
@@ -22,7 +22,7 @@
     let(:alternative_given_names) { "" }
     let(:alternative_family_name) { "" }
 
-    it { is_expected.to be true }
+    it { is_expected.to be false }
```

Comment:
It makes sense that this would now be `false`, however we now don't have a test in this file which considers the case when the form is valid, so I think we should add one.

We should probably include a test where `has_alternative_name` is `true` and one where `has_alternative_name` is `false` too.

Reply:
Thanks to the conditional validation, i think it is now true once again. 
Ill add the two new tests however.